source/environments/Pict-Environment-Object.js

```diff
@@ -203,6 +218,41 @@ class PictEnvironmentObject
 
 		return true;
 	}
+
+	customReadClassFunction (pAddress, pClass)
+	{
+		let tmpContent = this.contentMap._CLASS_LIST.indexOf(pClass) > -1 ? true : false;
+
+		if (this.storeEventLog) this.eventLog.Assign.push(this.createEventLogEntry(pAddress, pClass));
```

Comment:
don't know how Steven feels about these, but I strongly avoid if statements with no brackets.

Reply:
These were copied from his code, but I don't mind changing it if we want consistency

Reply:
Oh then that's fine. I'd rather adhere to the norm of the repo